### ISSUE TITLE
[website] old previews still prominently served and found by search engines

### ISSUE THREAD

**Issue by bar-g** [oilshell/oil#1781]
A quite confusing example of a top search engine result for "oilshell funcs" is https://www.oilshell.org/preview/doc/oil-proc-func-block.html e.g. still talking about "do" not "call" etc.

A more recent page, but I'm unsure if still missing something from the old page, however only found from "All Docs" page, is:
https://www.oilshell.org/release/latest/doc/proc-func.html

**Comment by andychu** [comment]
Yeah I've noticed this too, I think it has to do with me posting some previews on Reddit

This page is supposed to help people find docs:

https://www.oilshell.org/release/0.19.0/doc/faq-doc.html


---

There might be a way to specify the canonical version to search engines   ... though I wonder if it works when you want to only modify the NEWER content to announce itself over the old

**Comment by bar-g** [comment]
One part may be something like disallowing previews/* in robots.txt?

Maybe also the /<version>/* doc directories, with the exception of /latest/*?

**Comment by andychu** [comment]
Yeah we can probably do that ... I have wanted to set up our own search

https://pagefind.app/

and then it will be easier to control what is indexed
